activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -305,9 +305,9 @@ def create_table(table_name, comment: nil, **options)
       #   # Creates a table called 'assemblies_parts' with no id.
       #   create_join_table(:assemblies, :parts)
       #
-      # You can pass a +options+ hash can include the following keys:
```

Comment:
Lets keep the original one and fix `a` to `an`. There is no reference of `options` upto this point, hence this form of mention.